common/src/thrift/block_master.thrift

```diff
@@ -15,49 +15,69 @@ struct WorkerInfo {
 
 service BlockMasterService extends common.TachyonService {
 
-  // Tachyon Client API
-
-  /*
+  /**
    * Returns the block information for the given block id.
+   * @param blockId
```

Comment:
I would suggest to remove the `@param`, `@throws` and `@return` annotations altogether ... the reason for that is that Thrift is not smart enough to respect these annotations and it ends up generating its own annotations ... the generated Java sources then end up having duplicate annotations ... for instance:

```
+    /**
+     * Returns the block information for the given block id.
+     * @param blockId
+     * @return the block information for the given block id
+     * @throws TachyonTException
+     * 
+     * @param blockId
+     */
```

which is not great